workflow/scripts/compute_gene_tf.py

```diff
@@ -91,7 +91,10 @@ def main(args):
     gp['peaks'] = gp.index.values
     gp['peaks'] = pd.Categorical(gp['peaks'], atac_sc_ad.var_names)
     gp['gene'] = pd.Categorical(gp['gene'], rna_sc_ad.var_names)
-
+    
+    # Binarize gene peak correlations
+    gp['cor'] = (gp['cor'] > 0).astype(int)
```

Comment:
@Huangy57  Should this `>0` or `!=0` ?

Reply:
All the negative correlated peaks are filtered out, so `>0` or `!=0` will be the same. I have tried different ways to binarize the gene peak correlations, so we can discuss more during our meeting